src/templates/drizzle/logo.hbs

```diff
@@ -1 +1,7 @@
-Drizzle
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 60 60" width="30" fill="currentColor" class="drizzle-u-marginRight">
```

Comment:
I would also add a `height` attribute. Some browsers are dumb and assume defaults of `300px` by `150px` if one or neither of these are undefined.

Also, the `xmlns` attribute is unnecessary when embedded in an HTML5 document.